org.adempiere.base/src/org/compiere/tools/FileUtil.java

```diff
@@ -486,11 +481,65 @@ public static File createTempFile(String prefix, String suffix, File directory)
 
         return f;	
 	}
+    
+    public static String getTempFolderName(File directory) throws IOException {

```

Comment:
please add javadoc and explanation needed for the optional directory parameter.

Reply:
Added